geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala

```diff
@@ -35,7 +36,19 @@ class HBaseDataStoreFactory extends DataStoreFactorySpi with LazyLogging {
 
   // TODO: investigate multiple HBase connections per jvm
   private lazy val globalConnection: Connection = {
-    val ret = ConnectionFactory.createConnection(HBaseConfiguration.create())
+    val conf = HBaseConfiguration.create()
+    val auth = conf.get("hbase.security.authentication")
```

Comment:
In the documentation, you mentioned that this happens in a kerberized environment.  Does that imply that a block like the following is in the hbase-site.xml?

<property>
   <name>hbase.security.authentication</name>
   <value>kerberos</value>
</property>

Basically, I'm wondering if that's how the configuration key-value pair is or if code sets it elsewhere?  (If it is set in configuration, is it worth explaining that in the documentation?  It may have been handled auto-magically by HDP.)